src/main/res/values/strings_pref.xml

```diff
@@ -84,7 +84,7 @@
     <string name="pref_sensors_orient_filter_medium">Medium filter</string>
     <string name="pref_sensors_orient_filter_no_filter">No filtration</string>
     <string name="pref_statusbar">Statusbar Icon</string>
-    <string name="pref_statusbar_desc">Show icon in statusbar</string>
+    <string name="pref_statusbar_desc">Show icon in statusbar (If this option is not available, the statusbar is forced by an other seting.)</string>
```

Comment:
Better:
... (If this option is not available, the status bar is forced by another setting)

Reply:
Removed the braces and it's text.